app/javascript/components/util/EligibilityTooltip.js

```diff
@@ -1,136 +1,62 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
+import _ from 'lodash';
 
 import { formatTimestamp } from '../../utils/DateTime';
 import ReactTooltip from 'react-tooltip';
 
+const eligibility_options = [
+  { conditional: 'household', icon: 'fa-house-user', title: 'In a household:', messageCount: 1 },
+  { conditional: 'eligible', icon: 'fa-comment', title: 'Eligible for notifications:', messageCount: 1 },
+  { conditional: 'reported', icon: 'fa-comments', title: 'Received a report today:', messageCount: 'all' },
+  { conditional: 'sent', icon: 'fa-comment-dots', title: 'Waiting for a response:', messageCount: 'all' },
+  // This bottom entry is the default. If none of the `conditional` fields are truthy within props.report_eligibility, the bottom will render
+  { conditional: '', icon: 'fa-comment-slash', title: 'Not eligible for notifications:', messageCount: 'all' },
+];
+
 class EligibilityTooltip extends React.Component {
   constructor(props) {
     super(props);
+    let ativeOption = eligibility_options.find(eo => props.report_eligibility[`${eo.conditional}`]) || _.last(eligibility_options);
+    ativeOption.messages = ativeOption.messageCount === 1 ? [this.props.report_eligibility.messages[0]] : this.props.report_eligibility.messages;
+    this.state = {
+      ativeOption,
```

Comment:
might be a stupid question but what does this variable mean?

Reply:
This component has several options right? You can be in a Household, or Not Eligible, or Reported or whatever. Based off props, you are only one thing. Previously, that had been determined by a bunch of `else-ifs` in the `render()`. Now, it determines which "option" you are in the `constructor()`.

Reply:
Oh, if you are referring to `ative`, it's because I use a macbook with a broken butterfly keyboard. That word should be `active`.

Reply:
yes that's exactly what I meant :joy: